pkg/pool-linear/contracts/LinearPoolRebalancer.sol

```diff
@@ -24,6 +24,8 @@ import "@balancer-labs/v2-interfaces/contracts/pool-linear/ILinearPool.sol";
 import "@balancer-labs/v2-solidity-utils/contracts/math/FixedPoint.sol";
```

Comment:
Probably changes in this file should just be ignored/reverted. You can switch your branch to main, copy this file and paste on this branch.